bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -1288,6 +1289,33 @@ public ClientConfiguration setEnsemblePlacementPolicySlowBookies(boolean enabled
         return this;
     }
 
+    /**
+     * Whether to enable BookieAddressResolver.
+     *
+     * @return flag to enable/disable BookieAddressResolver.
+     */
+    public boolean getEnableBookieAddressResolver() {
```

Comment:
@StevenLuMT The `bookieAddressResolverEnabled` added this time is a client-side setting, so it cannot be set for `ServerConfiguration`. I replaced `ServerConfiguration` with `ClientConfiguration` because I want to test the behavior of `bookieAddressResolverEnabled` when it is true and when it is false.

Also, I think that `ClientConfiguration` is more appropriate than `ServerConfiguration` to pass as the first argument of `DiscoveryCommand#apply()`.
https://github.com/apache/bookkeeper/blob/62cfc0ec8670cb66ce4bf2b0d707f42cba5e3354/bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/helpers/DiscoveryCommand.java#L49

Even if `ServerConfiguration` is passed, it will be converted to `ClientConfiguration` internally.
https://github.com/apache/bookkeeper/blob/62cfc0ec8670cb66ce4bf2b0d707f42cba5e3354/bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/helpers/ClientCommand.java#L57-L61

Reply:
@StevenLuMT Renamed to "getBookieAddressResolverEnable**d**". What do you think?